full_import.sh

```diff
@@ -1,9 +1,21 @@
 #!/bin/bash
 
-until host ${1} &> /dev/null
+retryCount=0
+
+# run the script as sudo so that you
```

Comment:
This will relaunch the script with sudo so it does not prompt you for the sudo password partway down the script.